### PR TITLE
changed input element properties

### PR DESCRIPTION
**Pull request by raghav1030**
changed the mouse pointer behavior when hovered and focused on input elements in login form.

### CODE REVIEW

File: src/pages/Login.jsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ import Logo from "../assets/logo.png";
 import { faRightToBracket } from "@fortawesome/free-solid-svg-icons";
 import { useNavigate } from "react-router-dom";
 import { useSnackbar } from "notistack";
+import './Login.css'
```
Comment (narayan954) [inline src/pages/Login.jsx:13]: if it's a common css for both file, you may name it index.css or style.css.

File: src/pages/Login.jsx [hunk 2]
```diff
@@ -120,6 +121,8 @@ const LoginScreen = () => {
               boxShadow: "0 0 20px rgba(104, 85, 224, 0.2)",
               borderRadius: "4px",
               // padding: "10px",
+
```
Comment (narayan954) [inline src/pages/Login.jsx:124]: format the code as well

**Review by narayan954 — CHANGES_REQUESTED** [review]
needs some minor changes

**Comment by narayan954** [comment]
mention the issue number you're solving as well in the pr so it can be linked

**Comment by narayan954** [comment]
@raghav1030  are you attending this issue?

**Comment by raghav1030** [comment]
Yes I'll be making changes  into it.

Actually my end sems were going on so I was a bit inactive here.

**Comment by narayan954** [comment]
> Yes I'll be making changes into it.
> 
> Actually my end sems were going on so I was a bit inactive here.

ohhh alright then

**Review by narayan954 — APPROVED** [review]
looks good to me!

**Comment by narayan954** [comment]
Thanks for your contribution! :)

**Pull request merged**